layouts/partials/openapi/render-object-table.html

```diff
@@ -148,3 +160,14 @@
     {{ end }}
     {{ return $type }}
 {{ end }}
+
+{{/*
+    Computes the description to display for a property.
```

Comment:
could you document the parameters please? (ie, a dict containing `property` and `required`)